src/js/ObjectClasses/Items/Shooter/Shooter.js

```diff
@@ -9,9 +10,19 @@ export class Shooter extends Actor {
     }
 
     onInitialize(engine) {
+        this.game=engine
```

Comment:
why are we using this.game instead of just engine? I would also prefer if we used a similar constructor for shooter as for player or other items, for the sake of generalisation